src/sinks/aws_cloudwatch_logs.rs

```diff
@@ -215,6 +320,7 @@ impl Service<Vec<Event>> for CloudwatchLogsSvc {
                 let (tx, rx) = oneshot::channel();
                 self.state = State::Put(rx);
 
+                debug!(message = "Submitting events.", amount_of_events = req.len());
```

Comment:
`amount_of_events` is weird. How about just `count`?

Reply:
naming is hard..